frontend/src/locale/index.ts

```diff
@@ -46,8 +46,14 @@ export const messages = {
     "profile.status": "Status",
     "profile.price": "Price",
     "profile.manageOrder": "Manage order",
-    "profile.items": "Tickets",
+    "profile.myTickets": "My Tickets",
+    "profile.myTickets.notickets":
+      "No tickets assigned to your email, you can buy one {linkTicket}",
```

Comment:
another thing that can help here is to show the email and say something like this:

```suggestion
    "profile.myTickets.notickets":
      "No tickets found assigned to your email: {email}. Make sure the attendee email and your email matches in your order or buy a ticket {linkTicket}",
```

so users know they need to update the attendee email to match the tickets